src/css/products.css

```diff
@@ -0,0 +1,241 @@
+@import url("https://fonts.googleapis.com/css?family=Dosis:400,600,700");
```

Comment:
why are these files css ? and lying outside the component heirachy

Reply:
These all are generated files. I have removed them and created a single css file main.css.